erica/api/v2/responses/model.py

```diff
@@ -0,0 +1,9 @@
+from erica.request_processing.erica_input import ErrorRequestQueue, ResponseGetFromQueue
+
+response_model_post_to_queue = {
+    201: {"description": "Job was successfully submitted to the queue and the job id was returned."},
+    422: {"model": ErrorRequestQueue, "description": "Job could not be submitted to the queue."}}
+
+response_model_get_from_queue = {
+    200: {"model": ResponseGetFromQueue, "description": "Job status was successfully retrieved from the queue."},
+    500: {"model": ErrorRequestQueue, "description": "Job status could not be retrieved from the queue."}}
```

Comment:
How is this used?

Reply:
Those responses are used in every post/get route for the job queue and define how the returned structure for the specified HTTP code should look like. This is necessary for the docs to display them. They are addressed with the "responses" attribute of the route

Reply:
Ok, I just do not understand how they are referenced

Reply:
With the responses attribute of the route:
`@router.post('/est_validations', status_code=status.HTTP_201_CREATED, responses=response_model_post_to_queue)`

Reply:
Somehow, I overlooked that. Thank you 🙈